packages/paste-website/src/pages/icon-system/how-to-add-an-icon.mdx

```diff
@@ -0,0 +1,134 @@
+  {
+    mdx(fields: {slug: {eq: "/icon-system/how-to-add-an-icon/"}}) {
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<contentwrapper>
+<TableOfContents headings={props.data.mdx.headings} />
+<content>
+
+<h1>{props.pageContext.frontmatter.title}</h1>
+Written for Sketch 59.1
+
+---
+
+## Finding an icon
+
+1. Make sure the icon you need doesn’t already live in the [Paste repo](https://twilio-labs.github.io/paste/?path=/story/overview-icons--list) 
+and that you can’t use one of the existing ones.
+2. Find a new icon by searching through the [Streamline app](https://app.streamlineicons.com/streamline-regular), 
+then message #help-design-system for the Sketch file.
+3. Create a branch in Abstract.
+    1. If you’re planning on adding the icon to Paste, first [create a Github issue](https://github.com/twilio-labs/paste/issues). 
+  Explain what you’re using it for, and list other potential use cases outside of your own. 
+  Add a link or screenshot of the proposed icon. If the team agrees, create a branch in 
+  the ["Paste icons" Abstract project](https://share.goabstract.com/0bdb9ae0-d1da-4021-80d3-338683d27b42).
+    2. If the icon won’t be added to Paste, create a branch in your own Abstract project 
+  and skip to [Using a Streamline icon](/icon-system/how-to-add-an-icon#using-a-streamline-icon).
+
+
+## Setting up an icon in Paste Icons
+
+1. Open “paste icons.sketch".
+2. Go to the page “Symbols".
+3. Duplicate the empty template artboards ("[16/icon name](https://share.goabstract.com/26080d38-a95a-46b1-a0ad-aa4ddba4abb3)",
+"[20/icon name](https://share.goabstract.com/96294d4e-50cc-4c7e-9cb9-b82a045fe970)", and "[24/icon name](https://share.goabstract.com/f51d0d18-67ce-4c02-8c44-5f8d390863d7)"). 
+Make sure the artboards are aligned evenly on the canvas with the existing icon artboards.
+4. Turn the new artboards into symbols.
+5. In the artboard names, replace “icon name" with the name of your icon.
+
+
+## Using a Streamline icon
+
+1. Copy and paste the Streamline icon onto the 24px artboard.
+2. Select all the layers and change the border width to 2.
+![Example of border width set to 2](../../assets/images/icon-guide/using-streamline-icons-1.png)
+3. Select the layer group. Lock the width-height aspect ratio. If the icon appears 
+larger than the bounds of the artboard, shrink the icon down to just fill the 
+artboard. 20–22 px usually works.
+![Example of locked width and height values](../../assets/images/icon-guide/using-streamline-icons-2.png)
+4. Duplicate the layer group. Rename the original group to “source" and hide it.
+5. Ungroup the new layer group. You’ll be working with these layers.
+![Sample layer groups](../../assets/images/icon-guide/using-streamline-icons-3.png)
+
+
+## Formatting an icon
+
+1. Convert your icons to outlines so that the width of the lines stay consistent.
+    1. Select all the layers and press “SHIFT + COMMAND + O", or use Sketch's menu, “Layer > Convert to Outlines".
```

Comment:
```suggestion
    1. Select all the layers and press "SHIFT + COMMAND + O", or use Sketch's menu, “Layer > Convert to Outlines".
```

There's a few of these fancy quotes below as well. Not a show stopper at all. Should I call them out?

Reply:
Totally worth.  Fixed, thanks!